README.md

```diff
@@ -1025,9 +1072,12 @@ sending us your pull request please run full Maven build:
 ```
 $ mvn clean install -Pqulice
 ```
-
 To avoid build errors use maven 3.2+.
 
+<!-- @todo #790:30min Complete instructions for Gradle build writing -->
```

Comment:
@paulodamaso this task is not looks like a `todo` for origin issue (#790) where was requested only to "document how to use framework within gradle", but you're requesting here to support gradle build/test for Takes itself. It would be more correct to submit a request for that as a separate ticket.

Reply:
@g4s8 Done, look at https://github.com/yegor256/takes/issues/847